indico/modules/events/registration/util.py

```diff
@@ -286,8 +286,21 @@ class Meta:
         unknown = RAISE
 
 
-def make_registration_schema(regform, management=False, registration=None, captcha_required=False):
-    """Dynamically create a Marshmallow schema based on the registration form fields."""
+def make_registration_schema(
+    regform,
```

Comment:
```suggestion
    regform,
    *,
```

Good opportunity to make those args keyword-only (we already always pass them as keywords)